blaze-client/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <!-- encoders are assigned the type
+             ch.qos.logback.classic.encoder.PatternLayoutEncoder by default -->
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n${LOGBACK_EXCEPTION_PATTERN:-%throwable}</pattern>
+        </encoder>
+    </appender>
+
+    <root level="${LOGBACK_ROOT_LEVEL:-WARN}">
+        <appender-ref ref="STDOUT" />
+    </root>
+</configuration>
```

Comment:
Surprise this file is necessary.  This has a test->test dependency from testing, so why is this not on the classpath already?

Reply:
Right, I'd feel much more comfortable without this file.